src/boot/efifirmware.c

```diff
@@ -0,0 +1,60 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "efifirmware.h"
+#include "util.h"
+#include <endian.h>
+
+static bool efifw_validate_header(const void *blob, size_t uki_efifw_len,
+                                  uint32_t *fwid_off, uint32_t *payload_off) {
```

Comment:
our usual parameter indentation is to break after the `(` for the first time, and then stick to double indentation. see how your own pe_use_this_firmware() below does it.

Reply:
in systemd we have this rule that return params (i.e. pointer params we dereference for write only, never for read) should be named `ret_xyz`, see https://systemd.io/CODING_STYLE/

afaics the 3rd and fourth arg is one of those, hence should be named like that.